components/card.css

```diff
@@ -0,0 +1,68 @@
+.card-imgwrapper{
+    object-fit: cover;
+
+   
+}
+img{
```

Comment:
Suggestion

- Avoid directly styling the element if it's not necessary

Reply:
This is basically a general rule when Using Images I always want that Image should respect the parent's height and width, not its own height and size. So I have used styling on the Img element